docs/detections/api/rules/rules-api-create.asciidoc

```diff
@@ -415,6 +415,24 @@ must be an {es} date data type.
 
 |==============================================
 
+[[opt-fields-eql-create]]
+===== Optional fields for event correlation rules
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description
+
+|event_category_field |String
+|Contains the event classification, such as `process`, `file`, or `network`. This field is typically mapped as a field type in the keyword family. Defaults to the `event.category` ECS field.
+
+|tiebreaker_field |String
+|Sets a secondary field for sorting events (in ascending order) if they have the same timestamp.
```

Comment:
@nastasha-solomon Originally I agreed, but as I dug into this I don't think "oldest to newest" is fully accurate, because this setting could reference a field that isn't time-based. The Elasticsearch docs [recommend](https://www.elastic.co/guide/en/elasticsearch/reference/current/eql.html#eql-search-specify-a-sort-tiebreaker) using the `event.sequence` ECS field, which is not a datetime: it's just a sequence number that increments on each subsequent event.

Curious if we need to explain this further in docs, or further specify "ascending, _lexicographic_ order" as the UI copy says. My hunch is that, similar to [above](https://github.com/elastic/security-docs/pull/2061#discussion_r898411988), the user of these advanced settings will be somewhat familiar with how ES/EQL works with their data, and just "ascending" is fine — that's how it's explained in the Elasticsearch docs, which our docs link to.

<img width="345" alt="image" src="https://user-images.githubusercontent.com/87339667/174641164-d1101d23-3d62-4d03-b0fe-32d0c3756572.png">

Reply:
Ah, now it makes sense why the UI specifies **lexicographic** order too. In that case, you're right that "oldest to newest" isn't fully accurate. I suppose you could add "lexicographic order" to enforce consistency between the UI and docs. Unfortunately, I don't have suggestions for defining the term though. 😅 I don't think alphabetic order is entirely correct and I'm not familiar enough with the term to provide a correct and clear example. Perhaps the intended user for these docs knows and understands the term?

Reply:
Yeah I go back and forth on including "lexicographic" in docs, maybe others have thoughts? 

If we did include it, I'd lean toward not defining it, since it's not something specific to Elastic or our Security app, just a general technical concept, easily Googleable.

Reply:
Sounds good, I'll add it in the next commit...